src/screens/TopicMemberScreen/index.js

```diff
@@ -169,7 +188,7 @@ const TopicMemberScreen = () => {
         setHeaderHide(true);
       }
     },
-    [animatedHeight]
+    [coverPath]
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dependency array for `handleScrollEvent` has been changed from `[animatedHeight]` to `[coverPath]`. This change makes sense because `handleScrollEvent` now depends on `coverPath`, not `animatedHeight`.
```diff
- [animatedHeight]
+ [coverPath]
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->